### DISCUSSION TITLE
Add informational quickfix when an import could not be resolved

### DISCUSSION THREAD

**Discussion by luabud** [microsoft/pylance-release#3712] [Enhancement]
### Discussed in https://github.com/microsoft/pylance-release/discussions/3297

<div type='discussions-op-text'>

<sup>Originally posted by **karrtikr** September  6, 2022</sup>

![image](https://user-images.githubusercontent.com/13199757/188751345-979190f8-e9c8-44c6-af37-700bc91a69f9.png)

During a user study for testing getting started experience, 3 out of 4 users got stuck at this step. They went searching outside the tool (VSCode) on how to resolve the issue, which is when they got lost as they weren't sure if the solution applies to them.

It would be nice if we could provide an internal link here like Typescript does:

![image](https://user-images.githubusercontent.com/13199757/188752365-62081e43-e8fe-46ab-a733-44ce76819846.png)

about how one can generally resolve these errors. Same as https://github.com/microsoft/pylance-release/issues/2483.

cc/ @cwebster-99 @karthiknadig </div>

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
